test/sign_blob_test.sh

```diff
@@ -33,10 +33,9 @@ echo "Sign the blob with cosign first and upload to rekor"
 $COSIGN_CLI sign-blob --output-certificate blob.cert --output-signature blob.sig $BLOB
 
 echo "Verifying ..."
-$COSIGN_CLI verify-blob --signature blob.sig --cert blob.cert $BLOB
+$COSIGN_CLI verify-blob --signature blob.sig --cert blob.cert --certificate-identity '.*' --certificate-oidc-issuer '.*' $BLOB
```

Comment:
As I noted above, I think we shouldn't overload this flag to support regex, and should add a separate flag. Otherwise, you run into potential risks with a character representing both regex and a certain character (like a period, valid for emails)

Reply:
Done